src/raftstore/store/fsm/peer.rs

```diff
@@ -2238,8 +2333,12 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
         bind_term(&mut resp, term);
         if self.fsm.peer.propose(self.ctx, cb, msg, resp) {
             self.fsm.has_ready = true;
+            self.fsm.group_state = GroupState::Ordered;
+            self.register_raft_base_tick();
         }
 
+        self.register_pd_heartbeat_tick();
```

Comment:
seems we do not report read flow statistic in the ordered state if only a hot read region.

Reply:
I suggest to let pd worker to report to PD periodically. Pd worker is the place where all information is collected and raftstore has no idea what's going on on coprocessor or scheduler.